src/MissingTableHelper.php

```diff
@@ -3,17 +3,28 @@
 namespace Keboola\GenericExtractor;
 
 use Keboola\Csv\CsvFile;
+use Keboola\GenericExtractor\Configuration\Extractor;
 use Keboola\GenericExtractor\Exception\UserException;
 
 class MissingTableHelper
 {
-    public static function checkConfigs($configs, $dataDir)
+    public static function checkConfigs($configs, $dataDir, Extractor $configuration)
     {
         foreach ($configs as $config) {
+            $api = $configuration->getApi($config->getAttributes());
+
+            if (!empty($config->getAttribute('outputBucket'))) {
+                $outputBucket = $config->getAttribute('outputBucket');
+            } elseif ($config->getAttribute('id')) {
+                $outputBucket = 'ex-api-' . $api->getName() . "-" . $config->getAttribute('id');
```

Comment:
A v pripade, ze API `name` je empty? Tak ten $outputBucket bude mat tvar `ex-api-generic-configId`? Name bude `generic`?

Reply:
Uz to vidim https://github.com/keboola/generic-extractor/blob/master/src/Configuration/Api.php#L26

Reply:
Jasne, musi to byt stejne ako tu:
https://github.com/keboola/generic-extractor/blob/master/src/Executor.php#L85

Reply:
A co ten `__kbc_default` teda? Aha, to sa tu zas vynulluje:
https://github.com/keboola/generic-extractor/blob/master/src/Executor.php#L120
Hh, aby to nebolo take jednoduche.

Reply:
Jo ty si to vlastne nalinkoval v tej description :D

Reply:
ja se ti snazim ulehcit pracu a ty si to ani neprectes :D

Reply:
Nechcel som sa nechat ovplyvnit :D Takto sme obaja dosli k rovnakemu zaveru haha